pkg/api/admin_users.go

```diff
@@ -115,8 +115,14 @@ func AdminDeleteUser(c *models.ReqContext) {
 func AdminDisableUser(c *models.ReqContext) {
 	userID := c.ParamsInt64(":id")
 
-	disableCmd := models.DisableUserCommand{UserId: userID, IsDisabled: true}
+	// External users shouldn't be disabled from API
```

Comment:
I don't  understand this code. Could you explain it?

Reply:
That's correct. It took me a while to understand why it worked, the `err == nil` threw me for a loop :) I would probably change the compare on the error to `err != models.ErrUserNotFound` instead. But we could also potentially create a new query instead that is a bit more clear.

Reply:
Changed to 
```go
if err := bus.Dispatch(authInfoQuery); err != models.ErrUserNotFound {
```